algorithms/2-search-solutions/binary_search_iterative.rb

```diff
@@ -0,0 +1,17 @@
+def binary_search(sorted_array, value)
+  collection_size = sorted_array.length - 1
+  low = 0
+  high = collection_size
+
+  while low <= high
+    mid = (low + high) / 2
+    if sorted_array[mid] > value
+      high = mid - 1
+    elsif sorted_array[mid] < value
+      low = mid + 1
```

Comment:
I believe your pluses and minuses may be flopped, although this way should work anyway.